docs/SchemaMigrationGuide.md

```diff
@@ -1,53 +1,54 @@
+ The database schemas will evolve with each release of new features and editions of FHIR. The corresponding FHIR Server for Azure would need to be upgraded to the next/latest version of the schema, which may require migration of the SQL Server schema.
 
-## There are two ways to upgrade the schema
+The migration is a T-SQL script which alters the database. The script filename is encoded with the version. For example - The version is an incrementing integer.
+
+## Methods for upgrading
 1. Automatic schema upgrade
 
-2. Schema upgrade via schema migration tool
+2. Schema upgrade via SQL Server schema migration tool
 
-## 1. Automatic Schema Upgrade
-Here, the schema migration is performed automatically on the server startup.
-The scripts that upgrade schema are part of any upgrade package for the application and run automatically when the application launches.
+## 1. Automatic schema upgrade
+When using the automatic schema upgrade method, the SQL Server schema migration is performed automatically on the FHIR Server for Azure upgrade.
+The scripts which upgrade the SQL Server schema are part of any upgrade package for the web application and run automatically when the web application launches.
 
-In case of any errors during upgrade, the application fails to start and the upgrade script transaction is not committed.
+In case of any errors during upgrade, the web application fails to start and the upgrade script transaction is not committed.
 
 - ### Prerequisites
 
     - Application should have admin access to the database.
 
-- ### When might you choose this option
-    -  There is no specific DBA role - In this case, automatic schema upgrade is a good choice as it just requires the application with admin access to the database.
+- ### When might you choose this method?
 
     - Non-Production scenarios - The auto upgrade is a good option for dev/tests environments. This will automatically apply any pending migrations when the application launches. Hence, the testing of schema upgrade becomes part of the test cycle for the application.
 
-## 2. Upgrade via Schema migration tool
-Schema migration tool is a command-line utility that admins can run to perform schema migrations on demand.
+## 2. Upgrade via SQL Server schema migration tool
+The SQL Server schema migration tool is a command-line utility which admins can run to perform SQL Server schema migrations on demand.
 
-Schema upgrade scripts would still be part of any upgrade package for the application but the scripts will not run automatically on the server startup. In order to perform the schema upgrade, the schema admin would need to run the schema migration tool.
+The SQL Server schema upgrade scripts would still be part of any upgrade package for the web application but the scripts will not run automatically on the FHIR server for Azure upgrade. To perform the SQL Server schema upgrade, the schema admin would need to run the SQL Server schema migration tool.
```

Comment:
Sorry for all these nit picks! - but I think it would be good to capitalize the "S" for FHIR Server for Azure